R/emodnet_wfs.R

```diff
@@ -1,8 +1,63 @@
+emodnet_cache_file <- function() {
+  cache_dir <- rappdirs::user_cache_dir("EMODnetWFS")
+  if (!dir.exists(cache_dir)) {
+    dir.create(cache_dir, recursive = TRUE)
+  }
+
+  getOption("emodnet.services.csv") %||%
+    file.path(cache_dir, "services.csv")
+}
+
 .emodnet_wfs <- function() {
-  utils::read.csv(
-    system.file("services.csv", package = "EMODnetWFS"),
-    stringsAsFactors = FALSE
+  cache_file <- emodnet_cache_file()
+  # try reading from GitHub ----
+  url <- getOption("emodnet.services.url") %||%
```

Comment:
this and the other `getOption()` are escape hatches https://blog.r-hub.io/2023/01/23/code-switch-escape-hatch-test/

we could also maybe document them, but setting them because GitHub is down is really "advanced".

Reply:
They seem fine for me :)